package.json

```diff
@@ -14,7 +14,9 @@
         "i18next": "19.4.5",
         "immutable": "4.0.0-rc.12",
         "lodash": "4.17.19",
-        "query-string": "6.13.1"
+        "query-string": "6.13.1",
+        "ts-loader": "^8.0.4",
```

Comment:
Suggestion: We have been trying to use explicit versions whenever possible to avoid unexpected regressions